src/v/kafka/server/handlers/sync_group.cc

```diff
@@ -21,6 +21,7 @@
 
 namespace kafka {
 
+template<>
```

Comment:
These two-stage handlers now use the helper template so these methods are template specializations now, just as they are in the single-stage case.